packages/playground/src/components/widgets/widget-connection/widget-connection.tsx

```diff
@@ -2,6 +2,12 @@ import { Component, Element, Prop } from "@stencil/core";
 
 import NetworkTunnel from "../../../data/network";
 
+const NETWORK_NAMES = {
+  "1": "Ethereum (Mainnet)",
+  "3": "Ropsten (Private)",
+  "42": "Kovan (Testnet)",
+  "4": "Rinkeby (Private)"
```

Comment:
Not sure where `Private` terminology is coming from. They're all Testnets.

Reply:
You're right. For some odd reason, the Spanish translation only showed "Kovan" as a testnet, and the others as "Privada" (Private). I'll modify it accordingly.